README.md

```diff
@@ -19,9 +19,14 @@ ARCH=amd64 docker build -t kinvolk/calico-hostendpoint-controller:v0.0.2-amd64 -
 ARCH=arm64 docker build -t kinvolk/calico-hostendpoint-controller:v0.0.2-arm64 --build-arg ARCH . && docker push kinvolk/calico-hostendpoint-controller:v0.0.2-arm64
 ```
 
+To build an image for another architecture which is not your native OS architecture there are multiple ways.
+The most convenient one is to install the `qemu-user-static` package on your system, maybe followed by `sudo systemctl restart systemd-binfmt.service` to set up binary translation so that the Linux kernel can run binaries for other architectures.
+Afterwards, the only temporary change needed is to explicitly use the Docker image for the other architecture instead of the multiarch `debian:9` image in the `FROM` line of the `Dockerfile`. For our case, you need to use `arm64v8/debian:9` because there is no `debian:9-arm64`.
```

Comment:
```suggestion
The downside of this approach is that a base image for the non-native architecture must be specified in the `FROM` directive of the Dockerfile instead of an image like `debian:9`, which is a multiarch image. For example, for building a Debian-based ARM image on an x86-64 machine you need to use `arm64v8/debian:9` (because there is no `debian:9-arm64`).
```

Reply:
done